packages/@aws-cdk/aws-rds/lib/instance.ts

```diff
@@ -888,6 +902,13 @@ abstract class DatabaseInstanceSource extends DatabaseInstanceNew implements IDa
       engineVersion: props.engine.engineVersion?.fullVersion,
       licenseModel: props.licenseModel,
       timezone: props.timezone,
+      ...props.parameters && {
+        dbParameterGroupName: new ParameterGroup(this, 'ParameterGroup', {
+          engine: props.engine,
+          description: 'Auto-generated parameter group',
+          parameters: props.parameters,
+        }).bindToInstance({}).parameterGroupName,
+      },
```

Comment:
Same question here - I wonder why is this done here in `DatabaseInstanceSource`, and not in `DatabaseInstanceNew`, which handles `dbParameterGroupName` of the L1?

Reply:
see above

Reply:
Yep.